.travis.yml

```diff
@@ -10,5 +10,5 @@ before_install:
 script:
   - go test -v ./...
 after_success:
-  - gox -output "dist/{{.OS}}_{{.Arch}}_{{.Dir}}"
-  - ghr --username iangrunert --token $GITHUB_TOKEN --replace --prerelease --debug pre-release dist/
\ No newline at end of file
+  - scripts/build.sh
+  - ghr --username iangrunert --token $GITHUB_TOKEN --replace --prerelease --debug pre-release pkg/
```

Comment:
I prefer dist because pkg has meaning in Go (see the pkg directory in your $GOPATH).